freedomofpress/securedrop-export
# Printer preflight

# Description

* Added start-vm method for making sure the Export VM is running from external VMs. This is for both printing and transferring documents to a drive
* Added printer-preflight method for making sure only one supported printer is connected
* Added method to check if less than or more than one supported printer is connected and new error code to tell the client when more than one printer is connected: `ERROR_MULTIPLE_PRINTERS_FOUND`.

# Test

1. Build the export debian package from this branch and install it in sd-export-usb
2. Run the client and see that you get the correct screens for each case: 0 printers connected, 1 printer connected, and multiple printers connected.
